src/docs/source/includes/_deployment.md

```diff
@@ -503,8 +503,10 @@ helm chart is installed using settings to leverage cloud backing services.
     mushop-user-init-1-8k62c              0/1     Completed   0          33s
     ```
 
-1. Open a browser with the `EXTERNAL-IP` created during setup, **OR** `port-forward`
-directly to the `edge` service resource:
+>Note: if you installed Istio service mesh, you should see `2/2` in the `READY` column and `1/2` for the init pods and the assets deploy pod.
```

Comment:
Do you think that makes sense to explain why you see 2/2? Like saying that is because of sidecar...

Reply:
I mention sidecars in the Istio portion, but I can add it here as well if that makes sense